taxi_demo/summary.ipynb

```diff
@@ -0,0 +1,1050 @@
+{
```

Comment:
Either restart the kernel and re-run the whole notebook before checking it in, or clear all the output. It looks sketchy to have out of order cell input numbers.